doc/content/examples/example9.md

```diff
@@ -0,0 +1,89 @@
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Structure
+       caption=Acceleration response spectra of the fixed base structure subjected to the Ormsby Wavelet.
+
+### Site response analysis
+
+The input file for simulating the individual soil response is presented below. The uniform soil domain has a shear wave velocity of $165$ $m/s$. Although the soil is uniform throughout, the `UniformLayerAuxKernel` is used to specify the top surface of the soil. This is necessary when using the `ComputeIsotropicElasticityTensorSoil` material. Alternatively, if the `ComputeIsotropicElasticityTensor` is used for a uniform soil, the `UniformLayerAuxKernel` is not necessary. In addition, periodic boundary conditions are applied to the two opposite faces of the soil domain which are perpendicular to the Z axis to ensure that they move in unison. This achieved using the `Periodic` option in `BCs`. The Ormsby Wavelet is applied at the bottom of the soil block and the response is recorded at the top. The 1% damped acceleration response spectrum at the top of the soil is presented below. The periods at which the response spectrum peaks are the fundamental periods of the soil domain. The periods inferred from this figure are very similar to those predicted by the theoretical solution $4H/((2n-1)~Vs)$, where $H$ is the depth of soil and $n$ is the index of the period of interest.
+
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Acceleration response spectra of the soil subjected to the Ormsby Wavelet.
+
+### Soil Structure Interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra at 10m height from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period. This amplification ratio shows a good agreement with that using LS-Dyna computed by [!citet](BolisettiThesis) as presented in [fig:SSI_Amp_Valid].
+
+!listing examples/ex09/SSI_Response.i
+
+!media media/examples/ex09_SSI.png
+       style=width:80%;margin-right:0px;float:center;
+       id=fig:SSI_Amp
+       caption=Amplification ratio of the structural response spectra from the free-field soil in SSI analysis. This is computed using MASTODON.
+
+!media media/examples/ex09_SSI_Valid.png
+      style=width:80%;margin-right:0px;float:center;
+      id=fig:SSI_Amp_Valid
+      caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+
+[fig:SSI_FreeField] presents the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis. It is seen that these to spectra are very similar.
```

Comment:
Anything other than 'identical' here will raise questions. Please specify where the differences are coming from.